editor.planx.uk/src/@planx/components/shared/FlagsSelect.tsx

```diff
@@ -0,0 +1,78 @@
+import { AutocompleteChangeReason, AutocompleteProps } from "@mui/material/Autocomplete";
+import Chip from "@mui/material/Chip";
+import ListItem from "@mui/material/ListItem";
+import { Flag, flatFlags } from "@opensystemslab/planx-core/types";
+import React, { useMemo } from "react";
+import InputRow from "ui/shared/InputRow";
+import { CustomCheckbox, SelectMultiple } from "ui/shared/SelectMultiple";
+
+interface Props {
+  value?: Array<Flag["value"]>;
```

Comment:
Annoying yes because `Flag["value"]` is optional, despite it always be populated/required in practice. Will try to do a more thorough revisit of types as a followup :+1: